docker-compose-web-modeler-beta.yaml

```diff
@@ -23,8 +23,6 @@ services:
   modeler-db:
     container_name: modeler-db
     image: postgres:14.2-alpine
-    ports:
-      - "5433:5432"
```

Comment:
Why I removed the exposed port: https://camunda.slack.com/archives/C6GBS7DKL/p1676474625791639